app/src/main/java/com/sunasterisk/appchat/db/firebase/client/AuthImpl.kt

```diff
@@ -1,28 +1,87 @@
 package com.sunasterisk.appchat.db.firebase.client
 
+import android.net.Uri
+import com.google.firebase.auth.AuthCredential
+import com.google.firebase.auth.AuthResult
 import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.auth.FirebaseAuthEmailException
+import com.google.firebase.firestore.CollectionReference
 import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.storage.FirebaseStorage
+import com.sunasterisk.appchat.db.Result
+import com.sunasterisk.appchat.db.entity.User
 import com.sunasterisk.appchat.db.firebase.RemoteConstant.EMAIL_NOT_VERIFIED
 import com.sunasterisk.appchat.db.firebase.RemoteConstant.MSG_EMAIL_NOT_VERIFIED
 import com.sunasterisk.appchat.db.firebase.service.AuthService
-import com.sunasterisk.appchat.utils.getOneShotResult
 import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
 import kotlinx.coroutines.tasks.await
+import java.util.*
```

Comment:
thường thì alt+ enter , ide sẽ import cả java.util.*, nhưng thực tế thì k cần import toàn bộ v mà chỉ cần class ta làm việc đến. K cần sửa cx đk nhé

Reply:
ok